conftest.py

```diff
@@ -0,0 +1,9 @@
+
+def pytest_addoption(parser):
+    parser.addoption("--evm-trace", dest='vmtrace', default=1, action='store_const', const=10, help="Run trace")
```

Comment:
I did consider using boolean here. But the idea was that in case, in the future, we decide to add multiple EVM traces with varying levels of details (the way it currently happens in GETH), we could use this number to determine which level requested by the user. That said, I think we can also implement boolean

Reply:
Ah, that makes perfect sense!